src/Gandi.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Nexylan packages.
+ *
+ * (c) Nexylan SAS <contact@nexylan.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Nexy\Gandi;
+
+use fXmlRpc\Client as FxmlrpcClient;
+use fXmlRpc\Proxy;
+use fXmlRpc\Transport\HttpAdapterTransport;
+use GuzzleHttp\Client as HttpClient;
```

Comment:
You should not use Guzzle directly. The HttpClient should be passed on the constructor.

It can be not passed, then in this case you may use the client discovery: http://docs.php-http.org/en/latest/discovery.html#http-client-discovery

Example: https://github.com/KnpLabs/php-github-api/blob/cacf6f38bf9e6c5242e2b6dc26a67c4791bc7751/lib/Github/Client.php#L125

In this case, they created a `HttpBuilder` but this is maybe overkill for our usage.

Reply:
BTW, the client should be automatically introduced on the service when httplug-bundle is used.